listenbrainz/webserver/errors.py

```diff
@@ -160,20 +160,20 @@ def not_found(error):
     def file_size_too_large(error):
         return handle_error(error, 413)
 
-    @app.errorhandler(500)
+    @app.errorhandler(InternalServerError)
```

Comment:
Why do we use `InternalServerError` here and the error codes elsewhere? Like why we don't use `@app.errorhandler(ServiceUnavailable)` instead of ` @app.errorhandler(503)` ?

Reply:
Is it because the unhandled error can only be an `InternalServerError`?

Reply:
You're right, we could make this more consistent.
I _believe_ that the other items use numbers instead of exceptions because that's how it always was. When I originally opened this PR with flask 0.12, you had to use an exception here instead of a numerical code, so I changed it. 
However, looking at the new documentation for flask 1.1 (https://flask.palletsprojects.com/en/1.1.x/errorhandling/#unhandled-exceptions, https://flask.palletsprojects.com/en/1.1.x/api/#flask.Flask.handle_exception) it seems that you can register it with a number. Let me test it and if it works I'll change it back for consistency

Reply:
Yes, in the end flask runs this errorhandler on unhandled exceptions even if we use 500, I changed it back for consistency, and added a comment explaining how this handler is different